docs/fundamentals/code-analysis/style-rules/ide0302.md

```diff
@@ -36,8 +36,8 @@ Options specify the behavior that you want the rule to enforce. For information
 | Property                 | Value                                     | Description                           |
 |--------------------------|-------------------------------------------|---------------------------------------|
 | **Option name**          | dotnet_style_prefer_collection_expression |                                       |
-| **Option values**        | `true`                                    | Prefer to use collection expressions. |
-|                          | `false`                                   | Disables the rule.                    |
+| **Option values**        | `true` &#124; `when_types_exactly_match`  | Prefer to use collection expressions. |
```

Comment:
@CyrusNajmabadi I'm guessing that `when_types_loosely_match` doesn't apply to IDE0302 (stackalloc)? I didn't see it in your tests either.